tests/module/utils/test_deprecation.py

```diff
@@ -14,8 +14,11 @@ def _get_caplog_record_msg(warn_caplog_manim):
 
 @pytest.fixture()
 def warn_caplog_manim(caplog):
-    caplog.set_level(logging.WARNING, logger="manim")
+    logger = logging.getLogger("manim")
+    logger.propagate = True
+    caplog.set_level(logging.INFO, logger="manim")
```

Comment:
```suggestion
    caplog.set_level(logging.WARNING, logger="manim")
```
?
Also, I'm not a fan of the repetition of the `logger` stuff, what do you think about making a `logger` fixture?

Reply:
the function you commented on already was a fixture 😄 but indeed, I did also add a `manim_caplog` one in my changes. the one defined here and mine do more or less the same, it is just the loglevel that is different.

I could refactor this file here to use `manim_caplog` as well. Any thoughts about that?

Reply:
Proposal to unify this and only use one custom caplog fixture: 5168c63e

Reply:
If the tests still pass, I'm all for it! I'll review the PR again later tonight. Hopefully, we can merge it soon :)